lib/place.ml

```diff
@@ -14,8 +14,11 @@ let normalize =
      Cette fonction normalize n'est utilisée que dans le module place
      la séquence [-–—] distingue les trois tirets possibles
   *)
-  let r = Str.regexp "^\\[\\([^]]+\\)\\] *[-–—] *\\(.*\\)" in
-  fun s -> Str.global_replace r "[\\1], \\2" s
+  let r = Str.regexp "^\\[\\([^]]+\\)\\] *[-] *\\(.*\\)" in
+  fun s -> 
+    let s = Mutil.replace_utf_8 s "–" "-" in
```

Comment:
Ça ne me plait pas de faire deux opérations de substitutions en plus.

Essaye (sans garantie) de simplement remplacer la regexp par
```
"^\\[\\([^]]+\\)\\] *\\(-|–|—\\) *\\(.*\\)"
```

Et plus loin
```
Str.global_replace r "[\\1], \\3" s
```

Reply:
Non, ca marche pas!

Reply:
```
"^\\[\\([^]]+\\)\\] *\\(-\\|–\\|—\\) *\\(.*\\)"
```

Reply:
It works.
I had to fight a bit in updateData.ml but I think I have it.